docs/management_api.md

```diff
@@ -43,7 +43,7 @@ you can choose between synchronous or asynchronous call to make sure initial wor
 The asynchronous call returns with HTTP code 202 before trying to create workers.
 
 ```bash
-curl -v -X POST "http://localhost:8081/models?initial_workers=1&synchronous=false&url=https://<s3_path>/squeezenet_v1.1.mar"
+curl -v -X POST "http://localhost:8081/models?initial_workers=1&synchronous=false&url=https://<cloud_OR_noncloud-endpoint>/squeezenet_v1.1.mar"
```

Comment:
@gunandrose4u This will be more like <cloud_OR_noncloud_storage> and not endpoint, please change